resources/views/authors/show.blade.php

```diff
@@ -1,29 +1,36 @@
 @extends('layouts.app')
 
 @section('content')
-    <div class="container">
-        <section class="gistlog-author-block">
-            <a href="https://github.com/{{ $author->username }}" class="profile-pic">
-                <img src="{{ $author->avatarUrl }}">
+    <div class="container mx-auto gistlog__container my-8">
```

Comment:
First thing I want to know when I see this is:

Are there `gistlog__container`s that aren't `container`? That aren't `mx-auto`? That aren't `my-8`? If not, then we should just apply all of those classes to `gistlog__container`

Reply:
`mx-auto` and `my-8` can be move to `gistlog__container`, on the other hand, when I try to move `.container` I get `(202:3) `@apply` cannot be used with .container because .container is nested inside of an at-rule (@media).`.

So, I'll leave the `.container` class here :)